devdoc/sm_requirements.md

```diff
@@ -0,0 +1,275 @@
+# State manager requirements
+================
+
+## Overview
+
+`State manager` (short:`sm`) is a module the manages the call state for the APIs of a module under the following sematics:
```

Comment:
>sematics [](start = 113, length = 8)

semantics

Reply:
semantics have been clarified.

---
In reply to: [423408272](https://github.com/Azure/azure-c-util/pull/37#discussion_r423408272) [](ancestors = 423408272)